storage/sub_query.go

```diff
@@ -0,0 +1,28 @@
+package storage
+
+type SubQuery int
+
+const (
+	QueryForAllLastOperationsPerResource SubQuery = iota
+	QueryForNonResourcelessOperations
+)
+
+var subQueries = map[SubQuery]string{
```

Comment:
maybe some documentation (comments) on how to work with sub queries? how to write the SQL, how it should be used etc.

Reply:
Done!